scripts/createDB.sql

```diff
@@ -130,9 +140,22 @@ CREATE TABLE `soen_390_db`.`manufacturing_order` (
   
   collate = utf8mb4_unicode_ci;
 
+-- Table to store the different orders of finished goods put in by customers.
+CREATE TABLE `soen_390_db`.`customer_order` (
+  `orderId` INT NOT NULL AUTO_INCREMENT,
+  `status` VARCHAR(45) NOT NULL,
+  `totalPrice` DECIMAL(10,2) NOT NULL,
```

Comment:
``totalPrice`` being a field in the table ``customer_order`` and not being obtained from a view that goes through the different ``customer_ordered_good`` would be awkward in the real world.

What if you needed to modify an order and remove a ``customer_ordered_good``? Would the developer have to manually modify the ``totalPrice`` field as well?

Assuming we're not allowing users to change an order once it has been created, this is fine.

Reply:
Yep I think we will only allow to cancel order but we cannot modify them